istio-control/istio-discovery/templates/enable-mesh-mtls.yaml

```diff
@@ -63,19 +63,6 @@ spec:
   - mtls:
       mode: PERMISSIVE
 ---
-apiVersion: networking.istio.io/v1alpha3
```

Comment:
Line 8 Destinationrule for API server does not exist in the old installer. I think we only need that if `global.mtls.enabled == true`, put it in the else branch, rather than in the top. Unless I miss anything new installer/operator specific.

Reply:
Fixed, thanks